kokoro/pypi.sh

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Code under repo is checked out to this directory.
+cd "${KOKORO_ARTIFACTS_DIR}/github/dataflux-pytorch"
+
+function setup_virtual_envs() {
+    sudo apt-get -y update
+
+    echo Setting up Python virtual environment.
+    sudo apt install -y python3.8-venv
+    python3 -m venv venv
+    source venv/bin/activate
+}
+
+function run_git_commands() {
+    echo Setting git permissions.
+    git config --global --add safe.directory "*" 
+
+    echo Installing git submodules.
+    git submodule init
+
+    echo Updating git submodules.
+    git submodule update
+}
+
+function install_requirements() {
+    echo Installing python3-pip.
+    sudo apt-get -y install python3-pip
+
+    echo Installing project from pypi
+    pip install gcs-torch-dataflux
+}
+
+function run_integration_tests(){
+    echo Running integration tests.
+    pytest dataflux_pytorch/integration_tests/integration_test.py -vv --junit-xml="${KOKORO_ARTIFACTS_DIR}/integration_tests/sponge_log.xml" --log-cli-level=DEBUG
```

Comment:
What about also running the unit tests?

Reply:
Currently the versions in pypi and in github are different. Using pypi version for unit test can fail frequently.

Reply:
Could you clarify? `Using pypi version for unit test can fail frequently.` to me sounds like if I'm a user who installs gcs-torch-dataflux from PyPI and run unit tests locally, the unit tests can fail? If so we need to debug what's going wrong and it's an indicator that we will need to include this path in the continuous tests.

Reply:
PyPi version is few changes behind github main branch. 
In the main branch, I can see a `/dataflux-pytorch/dataflux_pytorch/_helper.py` , while this does not exists in pypi version.

Reply:
oh sorry I misunderstood. yeah you are right. When you install the PyPI version you don't have "unit tests" in it as we didn't package it in the release, and we shouldn't do either.

So we only need to run the integration tests with the installed PyPI version, that's right.